src/domain/community/user/user.resolver.queries.ts

```diff
@@ -60,6 +62,41 @@ export class UserResolverQueries {
     return await this.userService.getUsers(limit, shuffle);
   }
 
+  @UseGuards(GraphqlGuard)
+  @Query(() => RelayStylePaginatedUser, {
+    nullable: false,
+    description: 'The users who have profiles on this platform',
+  })
+  @Profiling.api
+  async users2(
```

Comment:
users2 is not very descriptive, maybe paginatedUsers or something similar. Even if it is temporary, that query name is a bad practice.